controllers/masteruserrecord/sync.go

```diff
@@ -54,6 +54,7 @@ func (s *Synchronizer) synchronizeSpec() error {
 		s.memberUserAcc.Spec.UserAccountSpecBase = s.recordSpecUserAcc.Spec.UserAccountSpecBase
 		s.memberUserAcc.Spec.Disabled = s.record.Spec.Disabled
 		s.memberUserAcc.Spec.UserID = s.record.Spec.UserID
+		s.memberUserAcc.Spec.OriginalSub = s.record.Spec.OriginalSub
```

Comment:
Any tests for that?

Reply:
I assume it's covered by https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/test/useraccount/assertion.go#L69 too?

Reply:
Like above, it's tested in `TestSynchronizeSpec` which also uses `MatchMasterUserRecord`.